app/src/main/java/com/hedvig/app/feature/marketing/pickmarket/PickMarketScreen.kt

```diff
@@ -129,8 +133,8 @@ fun PickMarketScreen(
             }
           },
           header = stringResource(R.string.market_language_screen_market_label),
-          label = data.market?.label?.let { stringResource(it) },
-          enabled = true,
+          label = selectedMarket?.label?.let { stringResource(it) },
+          enabled = enabled,
```

Comment:
This was always true before, now it depends on if market and lanaguge are both not null as I think that's where this enabled comes from.
From the PR files maybe it's a bit hard for me to grasp how this entire flow goes, but is this expected behavior? I guess it was always true before to let the user pick a market if it ended up being null somehow (not sure how)

Reply:
Good catch, error on my part. This should always be enabled